.github/workflows/test.yml

```diff
@@ -375,6 +424,47 @@ jobs:
         env:
           EXPERIMENTAL_RUST_CODEPATH: true
 
+  turborepo_integration_rust_windows:
```

Comment:
It just needs to be non blocking, since rust is also non blocking I can do that. But if Rust becomes blocking before windows does, the whole matrix will fail and block

Reply:
done